.github/workflows/preDeploy.yml

```diff
@@ -30,6 +31,9 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
 
+      - name: Check if merged pull request was an automatic version bump PR
+        run: echo "::set-output name=IS_VERSION_BUMP_PR::${{ constains(steps.getMergedPullRequest.outputs.labels, automerge) && github.actor == 'OSBotify' }}"
```

Comment:
Are we expecting this to return `true`/`false`? Can we verify it works as intended via public repo? I have had issues with booleans

Reply:
Tested in public repo w/ code:

```yaml
jobs:
  echoActor:
    runs-on: macos-latest
    outputs:
      isActorRory: ${{ steps.isActorRory.outputs.IS_ACTOR_RORY }}
    steps:
      - name: Echo actor
        run: echo ${{ github.actor }}

      - name: Is Actor Rory?
        id: isActorRory
        run: echo "::set-output name=IS_ACTOR_RORY::${{ github.actor == 'roryabraham' && 'blah' == 'blah' }}"

  # This workflow contains a single job called "build"
  build:
    # The type of runner that the job will run on
    runs-on: macos-latest
    needs: echoActor
    if: ${{ needs.echoActor.outputs.isActorRory == 'true' }}

    # Steps represent a sequence of tasks that will be executed as part of the job
    steps:
      - name: Run a nested script
        run: echo "Hello $(echo 'World')!"

      # Runs a single command using the runners shell
      - name: Run a one-line script
        run: echo ${{ toJSON(github.event) }} && echo "I am Andrew" && exit 0
```

And as you can see in the [workflow run](https://github.com/Andrew-Test-Org/Public-Test-Repo/actions/runs/658456018/workflow), the `build` job is not skipped.

Reply:
We are comparing the result to string `'true'` and/or `'false'` throughout the rest of the workflow, as we've discovered is usually necessary